### PR TITLE
add support for georss

### LINKED ISSUE #6: add support for GeoRSS

### PR DESCRIPTION
**Pull request by jgravois**
resolves #6

### CODE REVIEW

File: src/lib/ogr-cmd.js [hunk 1]
```diff
@@ -37,7 +38,21 @@ function csvParams (cmd, options) {
...
+ * Add parameters specific to a GeoRSS export
+ *
+ * @param {array} cmd - an array of OGR command parts to modify
+ * @param {object} options - may contain fields
+ * @return {array}
+ * @private
+ */
+function georssParams (cmd, options) {
+  cmd.push('-dsco', 'USE_EXTENSIONS=YES')
+  console.log(cmd)
```
Comment (dmfenton) [inline src/lib/ogr-cmd.js]: please remove

**Review by dmfenton — CHANGES_REQUESTED** [review]
This looks great! Would you be willing to post the results of a geojson to georss conversion so we can eyeball the results?

**Comment by jgravois** [comment]
> Would you be willing to post the results of a geojson to georss conversion so we can eyeball the results?

of course!

``` xml
<?xml version="1.0"?>
<rss version="2.0" xmlns:georss="http://www.georss.org/georss">
  <channel>
    <title>title</title>
    <description>channel_description</description>
    <link>channel_link</link>
    <item>
      <title>Feature 1</title>
      <ogr:FID>1</ogr:FID>
      <ogr:rownum>1</ogr:rownum>
      <ogr:latitude>40</ogr:latitude>
      <ogr:longitude>-88</ogr:longitude>
      <quant:val>1002</quant:val>
      <expected:c>red</expected:c>
      <FID:1>1</FID:1>
      <georss:point>40 -88</georss:point>
    </item>
    <item>
      <title>Feature 2</title>
      <ogr:FID>2</ogr:FID>
      <ogr:rownum>2</ogr:rownum>
      <ogr:latitude>40</ogr:latitude>
      <ogr:longitude>-87.9</ogr:longitude>
      <quant:val>1003</quant:val>
      <expected:c>red</expected:c>
      <FID:1>2</FID:1>
      <georss:point>40.0 -87.9</georss:point>
    </item>
    <!-- many more -->
  </channel>
</rss>
```

File: src/lib/ogr-cmd.js (hunk 1 above)
Comment (jgravois) [reply src/lib/ogr-cmd.js]: whoops.  thought i caught that.

**Comment by jgravois** [comment]
should we populate the title, description and link fields?

``` xml
<title>koop</title>
<description>Transform, query, and download geospatial data on the web</description>
<link>http://koopjs.github.io/</link>
```

maybe even dig something relevant out of the input?

**Comment by dmfenton** [comment]
Good idea to support that if the options are passed in, but shouldn't be required

**Comment by ajturner** [comment]
@juliekdang - you may want to see this PR since you just asked yesterday.

**Review by dmfenton — APPROVED** [review]

**Comment by dmfenton** [comment]
@jgravois I'm going to merge this as is. Title, description and link can come next.

**Comment by jgravois** [comment]
sounds good to me!

**Pull request merged**